internal/config/config.go

```diff
@@ -123,11 +82,11 @@ func (c *cfg) Write() error {
 	return ghConfig.Write(c.cfg)
 }
 
-func (c *cfg) Aliases() *AliasConfig {
+func (c *cfg) Aliases() gh.AliasConfig {
 	return &AliasConfig{cfg: c.cfg}
 }
 
-func (c *cfg) Authentication() *AuthConfig {
+func (c *cfg) Authentication() gh.AuthConfig {
 	return &AuthConfig{cfg: c.cfg}
 }
```

Comment:
Is the dropped pointer intended or legal?

```suggestion
func (c *cfg) Authentication() *gh.AuthConfig {
	return &AuthConfig{cfg: c.cfg}
}
```